index.html

```diff
@@ -0,0 +1,692 @@
+     ></path>
+    </svg>
+   </a>
+        <nav class="nav">
+            <ul class="nav-list">
+                <li>
+                    <a class="subscribe" href="#">SUBSCRIBE NOW</a>
+                </li>
+                <li>
+                    <a class="login" href="#">LOG IN</a>
+                </li>
+            </ul>
+        </nav>
```

Comment:
the `<nav>` element  should wrap the content actually used for the entire navbar not just part of it.
The `<header>` instead could be used for the main picture with caption, see the inline comment  bellow :point_down: